packages/camera/camera_android_camerax/lib/src/preview.dart

```diff
@@ -17,6 +17,9 @@ import 'use_case.dart';
 @immutable
```

Comment:
Adds functionality to dynamically set target rotation of Preview instance.

P.S. Support for setting initial target rotation already supported :)